lib/LedgerNodeOperations.js

```diff
@@ -46,9 +48,37 @@ module.exports = class LedgerNodeOperations {
    * @return a Promise that resolves once the operation completes.
    */
   async validate(operation) {
+    if(!(operation && typeof operation === 'object')) {
+      throw new BedrockError('Operation must be an object.', 'TypeError');
```

Comment:
Same question here ... our pattern is to use `TypeError` for this sort of thing.